docs/examples/paint-brush.md

```diff
@@ -1,7 +1,8 @@
 # Paint Brush
 
-Generates random 'art' by drawing across the screen.
-Press ``||controller:A||`` to change the size of the brush.
+Creates a unique 'work of art' by painting random brush strokes across the screen.
+
+Press ``||controller:A||`` to switch the size of the brush from small to big.
```

Comment:
Oh, maybe say 'and back' or something to make it obvious it will go back and forth?

Reply:
ok, ok